components/GameDetailsView.jsx

```diff
@@ -0,0 +1,217 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import BackButton from "@/components/buttons/BackButton";
+import styled from "styled-components";
+import {
+  StyledButton,
+  StyledLinkButton,
+  StyledButtonWrapper,
+  StyledDeleteButton,
+} from "@/components/buttons/DefaultButtons";
+
+export default function GameDetailsView({
+  selectedGame,
+  onEditGame,
+  onDeleteGame,
+  gameId,
+}) {
+  const router = useRouter();
```

Comment:
This is something I also overlooked when we fixed the bug earlier today:

Now that you are working with `useRouter` directly in this component, you could destructure the `gameId` within this component and could get rid of the `gameId` prop:

```suggestion
  const router = useRouter();
  const { gameId } = router.query;
```